test/e2e/specs/change-detection.test.js

```diff
@@ -52,6 +57,19 @@ describe( 'Change detection', () => {
 		await page.setRequestInterception( false );
 	}
 
+	it( 'Should autosave post', async () => {
+		await page.type( '.editor-post-title__input', 'Hello World' );
+		const isSavingState = await page.waitForSelector( '.editor-post-saved-state.is-saving' );
```

Comment:
Is there a way you can think of that we could avoid sitting and waiting for the autosave to occur? Adds a significant delay to the runtime of E2E tests otherwise.

Not necessarily a blocker.

Reply:
In core we currently respect a config variable, `AUTOSAVE_INTERVAL` which gets localized as `autosaveL10n.autosaveInterval`. We probably want to respect the same [configuration option](https://codex.wordpress.org/Editing_wp-config.php#Modify_AutoSave_Interval) and it we follow the same pattern of localizing the setting, we could overwrite that here in our test to make it much shorter.

Reply:
Added support for `AUTOSAVE_INTERVAL` in https://github.com/WordPress/gutenberg/pull/6257/commits/d795e1db8ff040dced0afc6de6baace3521246ec

Reply:
It worries me a bit with race conditions. I do like the addition of respecting `AUTOSAVE_INTERVAL` constant though.

Could we just trigger the autosave programmatically?

```js
await page.evaluate( () => {
	dispatch( 'core/editor' ).doAutosave();
} );
```